ebpf_ffi/cbpf.cc

```diff
@@ -32,7 +32,16 @@ bool load_cbpf_program(void *prog_buff, size_t size, std::string *error,
   struct sock_fprog program;
   program.len = size;
   program.filter = insn;
-  if (setsockopt(socks[0], SOL_SOCKET, SO_ATTACH_FILTER, &program,
+  struct timeval tv;
```

Comment:
put a blank line before this one :)

Additionally, add a comment stating why we need a timeout (e.g the generated filter might drop the packets)

Reply:
Done